sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -137,6 +381,34 @@ private static void DisplayHelp()
             Console.WriteLine();
         }
 
+        /// <summary>
+        ///   Reads the selection of the application's user from the
+        ///   console for samples.
+        /// </summary>
+        ///
+        /// <param name="samples">The available samples.</param>
+        ///
+        /// <returns>The validated selection that was made.</returns>
+        ///
+        private static int? ReadSelection(IReadOnlyList<IEventHubsSample> samples)
+        {
+            return ReadSelection(samples.Count() + extraOptionsForSamples.Length);
```

Comment:
nit:  Since this is a list, you can use the `Count` property directly.   Not a huge deal since the LINQ method will delegate to it, but I'd still recommend it.